onert-micro/CMakeLists.txt

```diff
@@ -56,6 +56,25 @@ else()
     return()
 endif()
 
+if (USE_STATIC_ALLOC)
+    # TODO: enable it
+    message(STATUS "FAILED ONERT-MICRO is not support Static Memory Manager now")
```

Comment:
Q. Static MM here means TensorArena in tflite micro ?

Reply:
I think it is similar to it. Static MM has buffer and know how to allocate intermediate tensors during execution. It works only with intermediate tensors.